http_sig_examples.py

```diff
@@ -149,28 +149,37 @@
 -----END EC PRIVATE KEY-----
 """
 
-exampleRequestMessage = b"""POST /foo?param=Value&Pet=dog HTTP/1.1
+def addContentDigest(message, body):
+    b64Digest = base64.b64encode(SHA512.new(body).digest()).decode('utf-8')
+    contentDigest = "sha-512=:%s:" % b64Digest
```

Comment:
For consistency with the rest of the script, this should use the http_sfv library primitives to create the value instead of string substitution.